typescript/src/apis/iempty_activity_consumer_api.ts

```diff
@@ -44,13 +50,16 @@ export interface IEmptyActivityConsumerApi {
    * Correlation.
    *
    * @async
-   * @param identity      The requesting users identity.
-   * @param correlationId The ID of the Correlation for which to retrieve the
-   *                      EmptyActivities.
-   * @returns             A Promise, which resolves with the retrieved EmptyActivities,
-   *                      or rejects an error, in case the request failed.
-   *                      This can happen, if the Correlation was not found,
-   *                      or the user is not authorized to see it.
+   * @param   identity           The requesting users identity.
+   * @param   correlationId      The ID of the Correlation for which to
+   *                             retrieve the EmptyActivities.
+   * @returns                    A list of waiting EmptActivities for the given
+   *                             Correlation.
```

Comment:
```suggestion
   *                             Correlation.
```

is correlation a name? if not -> correlation.

Reply:
yes it is.